dv/riscv_compliance/README.md

```diff
@@ -19,13 +19,14 @@ How to run RISC-V Compliance on Ibex
    - Verilator
    - fusesoc
    - srecord (for `srec_cat`)
-   - A RV32 compiler
+   - A RV32 compiler    # Toolchain provided with ibex will work
```

Comment:
We do not provide a toolchain with Ibex. We have an unsupported toolchain that we use in our CI runs. Hence, can you please remove this comment?

Reply:
Ok